pp/integrations/plugin-packs/procedures/operatingsystems-linux-telegraf-agent.md

```diff
@@ -0,0 +1,1533 @@
+| swap.usage.percentage | %     |
+
+> To obtain this new metric format, include **--use-new-perfdata** in the **EXTRAOPTIONS** service macro.
+
+</TabItem>
+<TabItem value="Systemd-Journal" label="Systemd-Journal">
+
+| Metric name           | Unit  |
+|:----------------------|:------|
+| journal.entries.count | count |
+
+</TabItem>
+<TabItem value="Systemd-Sc-Status" label="Systemd-Sc-Status">
+
+| Metric name                    | Unit  |
+|:-------------------------------|:------|
+| systemd.services.running.count | count |
+| systemd.services.failed.count  | count |
+| systemd.services.dead.count    | count |
+| systemd.services.exited.count  | count |
+| *sc*#status                    | N/A   |
+
+> To obtain this new metric format, include **--use-new-perfdata** in the **EXTRAOPTIONS** service macro.
+
+</TabItem>
+<TabItem value="Traffic" label="Traffic">
+
+| Metric name                                     | Unit  |
+|:------------------------------------------------|:------|
+| *interface*#status                              | N/A   |
+| *interface*#interface.traffic.in.bitspersecond  | b/s   |
+| *interface*#interface.traffic.out.bitspersecond | b/s   |
+
+</TabItem>
+<TabItem value="Uptime" label="Uptime">
+
+| Metric name           | Unit  |
+|:----------------------|:------|
+| system.uptime.seconds | s     |
+
+</TabItem>
+</Tabs>
+
+## Prerequisites
+
+The prerequisites below have to be applied to the Linux servers to be monitored.
+
+### Network flow
+
+Two TCP flows must be open from the host to the poller.
+
+| Source | Destination    | Protocol | Port | Purpose |
+| ------ | -------------- | --------- | ---- | --- |
+| Monitored host | Poller | TCP       | 1443 | Access to the Telegraf agent's configuration. |
+| Monitored host | Poller | TCP       | 4317 | OpenTelemetry data flow. |
+
+### System prerequisites on the poller
+
+> To be able to use the Telegraf agent, you must use a poller with at least version `24.04.2` of `centreon-engine`. The Telegraf agent will configure itself via a HTTPS request sent to Centreon Engine.
+
+1. For this to work, you must first get a valid certificate or generate a self-signed one on the poller as detailed below.
+
+> In the command below, replace `${HOSTNAME}` with the poller's FQDN if they don't match.
+> If you set an IP address in the access parameters to the configuration server instead of an FQDN,
+> the Telegraf agent will refuse the certificate.
+
+```bash
+openssl req -new -subj "/CN=${HOSTNAME}" -addext "subjectAltName = DNS:${HOSTNAME}" -newkey rsa:2048 -sha256 -days 365 -nodes -x509 -keyout /etc/centreon-engine/conf-server.key -out /etc/centreon-engine/conf-server.crt
+chown centreon-engine: /etc/centreon-engine/conf-*
+```
+
+> The `-days 365` option limits the certificate validity to one year. You may choose a longer or shorter duration according to your security/maintainance preferences.
+
+2. Then define Engine's configuration server for Telegraf.
```

Comment:
```suggestion
2. Then provide Engine with the connection information it needs to give to the Telegraf agent so that it can send information to Engine.
```